backend_test.py

```
@@ -0,0 +1,331 @@
+            self.tests_passed += 1
+            print(f"✅ {name}")
+        else:
+            print(f"❌ {name}: {details}")
+            self.issues.append(f"{name}: {details}")
+
+    def read_file(self, file_path):
+        """Read file content safely"""
+        try:
+            with open(self.app_root / file_path, 'r') as f:
+                return f.read()
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)